src/buildstream/_cas/casremote.py

```diff
@@ -59,18 +59,7 @@ def _configure_protocols(self):
 
         local_cas = self.cascache.get_local_cas()
         request = local_cas_pb2.GetInstanceNameForRemotesRequest()
-        cas_endpoint = request.content_addressable_storage
-        cas_endpoint.url = self.spec.url
-        if self.spec.instance_name:
-            cas_endpoint.instance_name = self.spec.instance_name
-        if self.spec.server_cert:
-            cas_endpoint.server_cert = self.spec.server_cert
-        if self.spec.client_key:
-            cas_endpoint.client_key = self.spec.client_key
-        if self.spec.client_cert:
-            cas_endpoint.client_cert = self.spec.client_cert
-        if self.spec.keepalive_time is not None:
-            cas_endpoint.keepalive_time.FromSeconds(self.spec.keepalive_time)
+        request.content_addressable_storage.CopyFrom(self.spec.to_localcas_remote())
```

Comment:
I'm not sure what's the idiomatic thing to do with grpcio, but I feel it would be better to pass it like
```
self.spec.to_localcas_remote(request.content_addressable_storage)
```

Would that be possible? Is it something we'd like to do, or is the current way more idiomatic?